### PR TITLE
CLIENT: enable OpenAL by default and decrease initial sound volume

### PR DESCRIPTION
**Pull request by codemasher**

### CODE REVIEW

File: code/client/snd_main.c [hunk 1]
```diff
@@ -498,7 +498,7 @@ void S_Init( void )
 		Cmd_AddCommand( "s_stop", S_StopAllSounds );
 		Cmd_AddCommand( "s_info", S_SoundInfo );
 
-		cv = Cvar_Get( "s_useOpenAL", "0", CVAR_ARCHIVE );// ENTE doesn't like the openAL-sound
```
Comment (robo9k) [inline code/client/snd_main.c:501]: > ENTE doesn't like the openAL-sound

Is this not a concern anymore?

Comment (codemasher) [reply code/client/snd_main.c:501]: Given that this opinion is from more than 10-12 years ago i don't think this has any relevance anymore.

Comment (kai-li-wop) [reply code/client/snd_main.c:501]: As asked by @mgerhardy in the WoP meeting, I think ENTE expressed it is not a problem to switch to OpenAL. Correct me, if I got it wrong.

**Review by mgerhardy — APPROVED** [review]

File: code/client/snd_main.c (hunk 1 above)
Comment (mgerhardy) [reply code/client/snd_main.c:501]: Indeed - that's how I understood it, too.

**Comment by robo9k** [comment]
Is there any science behind the new default values for `s_volume` and `s_musicvolume` or are they arbitrary?

**Comment by codemasher** [comment]
The `s_volume` is set to 50% which i think is loud enough, `s_musicvolume` is arbitrary and may be subject to change (it should be lower than `s_volume`, though). 
For the science behind this: I've seen several streams now where, when the streamer started a new game, they were pleasantly surprised when the game didn't yell at them in full volume. Some even mentioned that the 50% volume (i can't recall which game it was) was a good measure.

**Review by kai-li-wop — APPROVED** [review]
There is a point in what @codemasher says. There is also Discord people are often using and lowering the volume of WoP because of it. I would give it a try and wait for feedback of the community.

**Pull request merged**